devicetypes/smartthings/ezex-smart-sub-meter.src/ezex-smart-sub-meter.groovy

```diff
@@ -0,0 +1,116 @@
+        multiAttributeTile(name:"power", type: "generic", width: 6, height: 4){
+            tileAttribute("device.power", key: "PRIMARY_CONTROL") {
+                attributeState("default", label:'${currentValue} W')
+            }
+            tileAttribute("device.energy", key: "SECONDARY_CONTROL") {
+                attributeState("default", label:'${currentValue} kWh')
+            }
+        }
+        standardTile("reset", "device.energy", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+            state "default", label:'reset kWh', action:"reset"
+        }
+        standardTile("refresh", "device.power", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+            state "default", label:'', action:"refresh.refresh", icon:"st.secondary.refresh"
+        }
+
+        main (["power", "energy"])
+        details(["power", "energy", "reset", "refresh"])
+    }
+}
+
+def parse(String description) {
+    log.debug "description is $description"
+    def event = zigbee.getEvent(description)
+    if (event) {
+        log.info event
+        sendEvent(event)
+    } else {
+        List result = []
+        def descMap = zigbee.parseDescriptionAsMap(description)
+        log.debug "Desc Map: $descMap"
+          
+        if (descMap.cluster == 0x0702 && descMap.attrId == 0x0000) {
+             log.debug "energy"
+             def map1 = [:]
+             map1.name = "energy"
+             map1.value = zigbee.convertHexToInt(descMap.value)/1000000
+                        
+             if (map1) {
+                  result << createEvent(map1)
+             }
+             log.debug "Parse returned $map1"
+        }
+               
+        List attrData = [[cluster: descMap.cluster ,attrId: descMap.attrId, value: descMap.value]]
+        descMap.additionalAttrs.each {
+            attrData << [cluster: descMap.cluster, attrId: it.attrId, value: it.value]
+        }
+        attrData.each {
+                def map = [:]
+                if (it.cluster == 0x0702 && it.attrId == 0x0400) {
+                        log.debug "meter"
+                        map.name = "power"
+                        map.value = zigbee.convertHexToInt(it.value)/1000
+                }
+                
+                if (map) {
+                        result << createEvent(map)
+                }
+                log.debug "Parse returned $map"
+        }
+        return result
+                
+    }
+  
+}
+
+/**
+ * PING is used by Device-Watch in attempt to reach the Device
+ * */
+def ping() {
+    return refresh()
+}
+
+def refresh() {
+    log.debug "refresh "
+    zigbee.electricMeasurementPowerRefresh() +
+           zigbee.simpleMeteringPowerRefresh()
+}
+
+def configure() {
+    // this device will send instantaneous demand and current summation delivered every 1 minute
+    sendEvent(name: "checkInterval", value: 2 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
```

Comment:
Device Health pings the device every `checkInterval-1` minutes. This would have it pinging the device every minute. This is too short. Even if the device _does_ report every minute, the `checkInterval` value should be the reporting interval * 2 + a couple of minutes of fuzzing, but honestly I think you don't need to do anything less than 12 minutes.